packages/esm-patient-chart-app/src/actions-buttons/stop-visit.component.tsx

```diff
@@ -23,14 +23,22 @@ const StopVisitOverflowMenuItem: React.FC<StopVisitOverflowMenuItemProps> = ({ p
         <button
           className="cds--overflow-menu-options__btn"
           role="menuitem"
-          title={`${t('endVisit', 'End visit')}`}
+          title={`${
+            !currentVisitIsRetrospective
+              ? t('endVisit', 'End visit')
+              : t('exitRetrospective', 'Exit retrospective entry')
+          }`}
           data-floating-menu-primary-focus
           onClick={openModal}
           style={{
             maxWidth: '100vw',
           }}
         >
-          <span className="cds--overflow-menu-options__option-content">{t('endVisit', 'End visit')}</span>
+          <span className="cds--overflow-menu-options__option-content">
+            {!currentVisitIsRetrospective
+              ? t('endVisit', 'End visit')
+              : t('exitRetrospective', 'Exit retrospective entry')}
+          </span>
```

Comment:
This is not correct.  Exiting retrospective entry should not be equated to ending a visit.  No ending of a visit should happen when a user exits retrospective entry.  The visit should either already be inactive/closed (because an end date was specified at the time of past visit creation), or should be active and left active (because the past visit was started in the past, and data needs to be entered from past dates/times, but is still open).